src/university_chatbot/university_conversation_logic_adapter.py

```diff
@@ -3,20 +3,24 @@
 
 import src.common_utils.language_utils.statement_utils as statement_utils
 from src.common_utils.language_utils.sentence_filter_utils import SentenceFilter
-
+import morfeusz2
 
 def find_best_tags_coverage(documents, tags):
     max_ratio = -1
     id_of_max_ratio_doc = -1
     tags_len = len(tags)
     for document in documents:
         tags_from_document = document['tags']
+        print('\n\ttags: ', tags)
+        print('\ttags_from_document: ', tags_from_document)
```

Comment:
I think it's better to avoid this debugging prints on master, it can make mess during presentation of how program is working